tests/integration/frameworks/test_keras_impl.py

```diff
@@ -54,7 +54,7 @@ def test_keras_save_load(model, kwargs, modelstore):
     if not TF2:
         # Initialize variables in the graph/model
         bentoml.keras._sess.run(tf.global_variables_initializer())
-        with bentoml.keras._default_sess():
+        with bentoml.keras._sess.as_default():
```

Comment:
is this line required for our users as well?

Reply:
it would be great to add some basic sample usage to the docstring

Reply:
for tf1 case then yes. i will add some docstring for that one.

Reply:
this can be accessed via `get_session()`